examples/torch/td3_halfcheetah.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""An example to train TD3 algorithm on InvertedDoublePendulum PyTorch."""
+import torch
+from torch.nn import functional as F
+
+# from garage.np.exploration_policies import AddGaussianNoise
+from garage import wrap_experiment
+from garage.envs import GymEnv, normalize
+from garage.experiment.deterministic import set_seed
+from garage.np.exploration_policies import AddGaussianNoise
+from garage.np.policies import UniformRandomPolicy
+from garage.replay_buffer import PathBuffer
+from garage.torch.algos import TD3
+from garage.torch.policies import DeterministicMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+from garage.trainer import TFTrainer
+
+
+@wrap_experiment(snapshot_mode='None')
+def td3_half_cheetah(ctxt=None, seed=1):
+    """Train TD3 with InvertedDoublePendulum-v2 environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+
+    """
+    set_seed(seed)
+    with TFTrainer(ctxt) as trainer:
```

Comment:
You shouldn't need to use `TFTrainer` in a torch algorithm, right? Please use `Trainer` instead to avoid confusing readers.